lib/IDE/CodeCompletionResultPrinter.cpp

```diff
@@ -187,6 +190,10 @@ class AnnotatingResultPrinter {
         printNestedGroup("callarg.type", i, e);
         continue;
       }
+      if (i->is(ChunkKind::CallArgumentDefaultBegin)) {
+        printNestedGroup("callarg.default", i, e);
+        continue;
+      }
```

Comment:
void printNestedGroup(StringRef tag, ChunkIter &i, const ChunkIter e) {
    assert(i != e && !i->hasText() &&
           CodeCompletionString::Chunk::chunkStartsNestedGroup(i->getKind()));
    auto nestingLevel = i->getNestingLevel();
    // Skip the "Begin" chunk.
    ++i;
    if (

    if (!tag.empty())
      OS << "<" << tag << ">";
    while (i != e && !i->endsPreviousNestedGroup(nestingLevel)) {

Reply:
Sure